mne/decoding/search_light.py

```diff
@@ -669,3 +703,22 @@ def _fix_auc(scoring, y):
                                  'two-class problems.')
             y = LabelEncoder().fit_transform(y)
     return y
+
+
+def _create_progressbar_context(inst, X, message):
+    if _check_verbose(inst.verbose):
+        multiply = (len(inst.estimators_)
+                    if isinstance(inst, GeneralizingEstimator) else 1)
+        n_steps = X.shape[-1] * max(1, multiply)
+        mesg = f'{message} {inst.__class__.__name__}'
+        context = ProgressBar(
+            n_steps, mesg=mesg, position=inst.position)
+    else:
+        context = contextlib.nullcontext()
+    return context
+
+
+def _check_verbose(verbose):
```

Comment:
Could you add a tiny docstring please?

Reply:
Yes, thanks for suggesting it. I sometimes need a small push to do the right thing. :)

Reply:
I can relate 😁